examples/basic/src/Button/Button.jsx

```diff
@@ -12,27 +14,60 @@ const ButtonStyled = styled.button`
   cursor: pointer;
   transition: .2s ease all;
   &:hover {
-    background-color: #6a76ef;
+    background-color: #6d6d6d;
   }
   ${props => props.outline && css`
+    border: 2px solid #3c3c3c;
+    background-color: white;
+    color: #3c3c3c;
+    &:hover {
+      background-color: #3c3c3c;
+      color: white;
+    }
+  `}
+
+  ${props => props.outline && props.variant === 'primary' && css`
     border: 2px solid #4756e6;
     background-color: white;
     color: #4756e6;
     &:hover {
-      background-color: #6a76ef;
+      background-color: #4756e6;
+      color: white;
+    }
+  `}
+
+  ${props => props.outline && props.variant === 'danger' && css`
+    border: 2px solid #ff5858;
+    background-color: white;
+    color: #ff5858;
+    &:hover {
+      background-color: #ff5858;
+      color: white;
+    }
+  `}
+
+  ${props => !props.outline && props.variant === 'primary' && css`
+    background-color: #4756e6;
+    color: white;
+  `}
+  ${props => !props.outline && props.variant === 'danger' && css`
+    background-color: #ff5858;
+    color: white;
+    &:hover {
+      background-color: #ff8585;
       color: white;
     }
   `}
 `;
 
-export const Button = ({ children, outline }) => (
-  <ButtonStyled outline={outline}>{children}</ButtonStyled>
+export const Button = ({ children, outline, variant }) => (
+  <ButtonStyled variant={variant} outline={outline}>{children}</ButtonStyled>
 );
 
 Button.propTypes = {
   outline: p.bool,
   disabled: p.bool,
   /** My description for loading prop */
```

Comment:
innecesary comment

Reply:
That comment is for the basic example and demonstrates the use of the Props component.

Reply:
The "tio Bob" is against this comment :smile: